components/hackathon/common/HackathonWinners.tsx

```diff
@@ -0,0 +1,90 @@
+import { Flex, Grid, Icon, Image, Link, VStack } from "@chakra-ui/react";
+import { FiExternalLink } from "react-icons/fi";
+import { Heading, Text } from "tw-components";
+
+interface IWinner {
+  name: string;
+  image: string;
+  link: string;
+  position: string;
+}
+
+interface HackathonWinnersProps {
+  winners: IWinner[];
+}
+
+export const HackathonWinners: React.FC<HackathonWinnersProps> = ({
+  winners,
+}) => {
+  return (
+    <VStack
+      mb={{
+        base: 40,
+        md: 0,
+      }}
+    >
+      <Heading size="title.2xl" textAlign="center" paddingTop={10}>
+        Hackathon Winners
+      </Heading>
+      <Grid
+        templateColumns={{
+          base: "repeat(1, 1fr)",
+          sm: "repeat(2, 1fr)",
+          md: "repeat(3, 1fr)",
+        }}
+        placeItems="center"
+        gap={10}
+        px={{
+          base: 10,
+          md: 20,
+        }}
+        py={10}
+      >
+        {winners.map(({ name, image, link, position }) => (
+          <Link href={link} isExternal key={name}>
```

Comment:
I wouldn't put a Flex and all that thing inside a Link component, you should use a LinkOverlay